applications/CompressiblePotentialFlowApplication/python_scripts/apply_far_field_process.py

```diff
@@ -1,6 +1,12 @@
 import KratosMultiphysics
-import KratosMultiphysics.CompressiblePotentialFlowApplication as CompressiblePotentialFlowApplication
-#from CompressiblePotentialFlowApplication import*
+import KratosMultiphysics.CompressiblePotentialFlowApplication as CPFApp
+import math
+
+def DotProduct(A,B):
```

Comment:
Better to do this sort of thing in c++.

Reply:
Do you know whether the dot product function is implemented in Kratos and exposed to python in somewhere like here: kratos/python/add_vector_to_python.cpp? I was looking for it but could not find it.